applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -32,11 +32,13 @@ def GetParametersFileName(self):
         return "ProjectParametersDEM.json"
 
     def GetInputParameters(self):
+        Logger.PrintWarning("DEM_analysis_stage.py", "Calls to this method (GetInputParameters) will become deprecated in the near future.")
         parameters_file_name = self.GetParametersFileName()
         parameters_file = open(parameters_file_name, 'r')
         return Parameters(parameters_file.read())
```

Comment:
I think we figured at some point that this can give problems
Better to use `with open()`

Reply:
Thanks Phil, that's true. That method is actually becoming deprecated. It can be improved using "with open" instead, but point is we are probably deleting the whole method very soon.